config.toml

```diff
@@ -35,294 +35,190 @@ time_format_blog = "2006-01-02"
-description = """
-Stefan's introduction to Flux and GitOps Toolkit at Rawkode Live.
-"""
-url         = "https://www.youtube.com/watch?v=HqTzuOBP0eY"
-
-[[params.resources.items]]
-title       = "The Power of GitOps with Flux v2 and GitOps Toolkit with Leigh Capili"
-description = """
-Leigh talks about getting started with Flux and GitOps Toolkit, reconciliation and observability, and the
-powerful use of Prometheus and Grafana.
-"""
-url         = "https://www.youtube.com/watch?v=0v5bjysXTL8"
-
-[[params.resources.items]]
-title       = " Introduction to GitOps and GitOps Toolkit by Stefan Prodan"
-description = """
-Alexis Richardson, CEO at Weaveworks and Stefan Prodan, maintainer of the Flux project introduce you to GitOps Toolkit.
-
-Talk from Cloud Native Nordics Tech Talks.
-"""
-url         = "https://www.youtube.com/watch?v=qQBtSkgl7tI"
-
-# Toolkit hero element
-[params.toolkit]
-title = "GitOps Toolkit"
-subtitle = """
-The set of APIs and controllers that make up the runtime for Flux.
-You can use the GitOps Toolkit to extend Flux, and to build your
-own systems for continuous delivery.
-"""
-image = "building-blocks.svg"
-
-[[params.toolkit.buttons]]
-text = "Get started"
-color = "dark"
-url = "https://toolkit.fluxcd.io/dev-guides/source-watcher/"
-
-[[params.toolkit.buttons]]
-text = "Learn more"
-url = "https://toolkit.fluxcd.io/components/"
-
-[params.family]
-title = "Other Flux projects"
-
-# Flagger hero element
-[params.flagger]
-title = "Flagger"
-subtitle = """
-Kubernetes Operator for the automation of promoting canary deployments using Istio,
-Linkerd, App Mesh, NGINX, Skipper, Contour, Gloo or Traefik routing for traffic
-shifting, and Prometheus metrics for canary analysis.
-
-The canary analysis can be extended with webhooks for running system
-integration/acceptance tests, load tests, or any other custom validation.
-"""
-image = "flagger-gitops.png"
-
-[[params.flagger.buttons]]
-text = "Get started"
-color = "dark"
-url = "https://docs.flagger.app/"
-
-[[params.flagger.buttons]]
-text = "Learn more"
-url = "https://flagger.app/"
-
-
 # Community hero element
 [params.community]
 title = "Community"
 summary = """
-The Flux project aspires to be the vendor-neutral home for GitOps in a Cloud Native world.<br>
-What we achieved up until today is only possible because of our community.
+The Racklet project aspires to engage a diverse set of people in the open source world, working
+towards a common goal of making data center software and hardware more accessible and secure.
 """
 
 [[params.community.highlights]]
 title = "GitHub Discussions"
 icon = "fab fa-github"
 text  = """
-Join the conversation in GitHub Discussions. Everything Flux v2 related ranging from specificiations and feature planning to Show & Tell happens here.
+We'd love to see you collaborate to this project! No contribution is too small, we value all input and perspectives.
```

Comment:
```suggestion
We'd love to see you collaborate in this project! No contribution is too small, we value all input and perspectives.
```

Reply:
I think it should be "collaborate to"?

Reply:
Typo on my side, should be "on", AFAIK it's "contribute to sth" and "collaborate on sth". Collaboration implies mutual communication, thus "on" to point out participation in the project/community (unlike contributing to something which can be a one-off external action). For clarity though the sentence could be written out in full:
```
We'd love to see you collaborate with us on this project!
```

Reply:
Suggestion changed to reflect that now.